centrallix/expression/exp_main.c

```diff
@@ -523,7 +523,7 @@ exp_internal_DumpExpression_r(pExpression this, int level)
 	    case EXPR_N_INTEGER: printf("INTEGER = %d", this->Integer); break;
 	    case EXPR_N_STRING: printf("STRING = <%s>", this->String); break;
 	    case EXPR_N_DOUBLE: printf("DOUBLE = %.2f", this->Types.Double); break;
-	    case EXPR_N_MONEY: printf("MONEY = %d.%4.4d", this->Types.Money.WholePart, this->Types.Money.FractionPart); break;
+	    case EXPR_N_MONEY: printf("MONEY = %4.4lld", this->Types.Money.Value); break;
```

Comment:
Are there tests covering this stuff?

Reply:
This function needs to be tested, probably using (expDumpExpression)[https://sourcegraph.com/github.com/LightSys/centrallix@1b27fd61a93915507c25d7170016112732b4c953/-/blob/centrallix/expression/exp_main.c#L578:1]